.circleci/config.yml

```diff
@@ -641,8 +682,7 @@ jobs:
       - run:
           name: Check if the test should run
           command: |
-            FILES_TO_CHECK="${PWD}/packages/celotool,${PWD}/packages/protocol,${PWD}/.circleci/config.yml"
-            ./scripts/ci_check_if_test_should_run_v2.sh ${FILES_TO_CHECK}
+            ./scripts/ci_check_if_test_should_run_v2.sh celotool
```

Comment:
For these e2e tests we should ensure that celotool has a dependency on the `protocol` package

Reply:
Added protocol as a dev dep of the celo tool